src/seedu/addressbook/Main.java

```diff
@@ -38,7 +42,11 @@ public static void main(String... launchArgs) {
     /** Runs the program until termination.  */
     public void run(String[] launchArgs) {
         start(launchArgs);
-        runCommandLoopUntilExitCommand();
+        try {
+			runCommandLoopUntilExitCommand();
```

Comment:
indentations are off. it is easy to follow coding style with eclipse auto-format